pkg/pbq/store/wal/metrics.go

```diff
@@ -0,0 +1,26 @@
+package wal
+
+import (
+	metricspkg "github.com/numaproj/numaflow/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var FilesCount = promauto.NewCounterVec(prometheus.CounterOpts{
+	Subsystem: "pbq_wal",
+	Name:      "wal_files_total",
+	Help:      "Total number of wal files/partitions",
+}, []string{metricspkg.LabelVertex})
+
+var EntriesCount = promauto.NewCounterVec(prometheus.CounterOpts{
+	Subsystem: "pbq_wal",
+	Name:      "total_entries_total",
+	Help:      "Total number of entries written across ALL wal files/partitions",
+}, []string{metricspkg.LabelVertex})
+
+var GarbageCollectingTime = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
You are going to add the remaining metrics as we discussed, right? :)

Reply:
Yes :)